recipes/llvm-core/all/conanfile.py

```diff
@@ -137,6 +211,11 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
             del self.options.with_libedit  # not supported on windows
+        if Version(self.version) < 18:
```

Comment:
Do you have a reference about zstd inclusion? Could please add a comment in the recipe, I did not find a reference in https://releases.llvm.org/18.1.0/projects/libcxx/docs/ReleaseNotes/18.html

Reply:
It's not explicit in the release notes. It's available since LLVM 15 - see https://reviews.llvm.org/D128465